modules/rabbit-support/src/main/scala/surge/rabbit/RabbitDataSource.scala

```diff
@@ -37,6 +37,8 @@ trait RabbitDataSource[Key, Value] extends DataSource[Key, Value] {
 
   def bufferSize: Int
 
+  val tracer: Tracer
```

Comment:
This may want a default no-op - or may actually get a default no-op tracer from the `DataSource` parent trait already